stylesheets/_component.rules.scss

```diff
@@ -298,3 +298,306 @@ td.table__form .rules + .form__group:last-of-type {
+        }
+    }
+
+    .rule-link {
+        position: relative;
+        padding-top: 10px;
+        padding-bottom: 10px;
+
+        &::after {
+            color: #dedede;
+            content: '\f061';
+            font-family: 'FontAwesome';
+            font-size: 19px;
+            left: 0;
+            position: absolute;
+            top: 17px;
+        }
+    }
+
+    .rule-link--or {
+        &::before {
+            content: '';
+            display: inline-block;
+            border-left: 4px solid color(gray, lightest);
+            margin-right: -5px;
+            height: 45px;
+            position: absolute;
+            top: -14px;
+            width: 0;
+        }
+
+        &::after {
+            left: 3px;
+        }
+
+        button {
+            background-color: lighten(#9d7ee3, 25);
+        }
+    }
+
+    .rule-block--else-if + .rule-links .rule-link--or,
+    .rule-block--and + .rule-links .rule-link--or {
+        border-left: 4px solid color(gray, lightest);
+        margin-left: 20px;
+        padding-left: 40px;
+
+        &::after {
+            left: 44px;
+        }
+    }
+
+    .rule-link--or + .rule-link--and {
+        padding-top: 0;
+
+        &::after {
+            top: 7px;
+        }
+    }
+
+    .rule-link--and {
+        border-left: 4px solid color(gray, lightest);
+        margin-left: 20px;
+
+        button {
+            background-color: lighten(#e8b750, 30);
+        }
+    }
+
+    .rule-link--else-if {
+        margin-left: -20px;
+        padding-top: 20px;
+        padding-bottom: 20px;
+
+        &::before {
+            border-top: 2px dashed color(gray, lightest);
+            content: '';
+            display: block;
+            width: 100%;
+            height: 3px;
+            position: absolute;
+            top: 38px;
+        }
+
+        &::after {
+            top: 27px;
+        }
+
+        button {
+            background-color: lighten(#3f8ccc, 40);
+            margin-left: 30px;
+            min-width: 76px;
+        }
+    }
+
+    .rule-block + .rule-block,
+    .rule-block--indent > .rule-block {
+        margin-top: -1px;
+
+        // visually distinguish blocks close together
+        .rule-block__header {
+            border-top: 1px solid rgba(255, 255, 255, .5);
+        }
+    }
+
+    // first block, indented one spot
+    .rule-block + .rule-block--return,
+    .rule-block--if + .rule-block--or {
+        margin-left: 40px;
+        width: calc(100% - 40px);
+    }
+
+    .rule-block--indent > .rule-links,
+    .rule-block--indent > .rule-block--indent + .rule-links {
+        padding-left: 20px;
+    }
+
+    .rule-block--indent {
+        border-left: 4px solid color(gray, lightest);
+        margin-left: 20px;
+        padding-left: 20px;
+    }
+
+    .rule-block__header {
+        border: 0;
+        box-shadow: none;
+        color: color(white);
+        font-family: $font-family-regular;
+        font-size: 18.66px; // For AA large text
```

Comment:
Perhaps an opportunity to bake the AA compliant sizes into the existing font-size variable set?

Reply:
+1 for that